stdlib/sqlite3/dbapi2.pyi

```diff
@@ -146,8 +159,23 @@ class Connection:
     total_changes: Any
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     def close(self) -> None: ...
+    if sys.version_info >= (3, 11):
+        def blobopen(self, __table: str, __column: str, __row: int, *, readonly: bool = ..., name: str = ...) -> Blob: ...
+
     def commit(self) -> None: ...
     def create_aggregate(self, name: str, n_arg: int, aggregate_class: Callable[[], _AggregateProtocol]) -> None: ...
+    if sys.version_info >= (3, 11):
+        # num_params determines how many params will be passed to the aggregate class. We provide an overload
+        # for the case where num_params = 1, which is expected to be the common case.
+        @overload
+        def create_window_function(
+            self, __name: str, __num_params: Literal[1], aggregate_class: Callable[[], _SingleParamWindowAggregateClass]
+        ) -> None: ...
+        @overload
+        def create_window_function(
+            self, __name: str, __num_params: int, aggregate_class: Callable[[], _WindowAggretateClass]
+        ) -> None: ...
```

Comment:
According to the documentation, `aggregate_class` can also be `None` to clear a window function:

```
@overload
def create_window_function(self, __name: str, __num_params: int, aggregate_class: None) -> None: ...
```

Reply:
Also, `__aggregate_class` in all three overloads.

Reply:
Good point, done